src/scss/vendors/_vendors.scss

```diff
@@ -83,7 +83,7 @@
 
 		a {
 			color: $navy-light;
-			font-size: rem(18);
+			font-size: rem(14);
```

Comment:
The UIO table of content should inherit the font size of the content wrapper. Please double check if this line is still needed. The same comment applies to its desktop css. Thanks.